faults/load/experiments.yaml

```diff
@@ -0,0 +1,85 @@
+---
+apiVersion: litmuschaos.io/v1alpha1
+description:
+  message: |
+    Execute the k6 engine using the JS script
+kind: ChaosExperiment
+metadata:
+  name: k6-loadgen
+  labels:
+    name: k6-loadgen
+    app.kubernetes.io/part-of: litmus
+    app.kubernetes.io/component: chaosexperiment
+    app.kubernetes.io/version: ci
+spec:
+  definition:
+    scope: Cluster
```

Comment:
I don't think we need `Cluster` scope here as all the operations are scoped to the chaos namespace itself, so `Namespaced` should suffice.

Reply:
fixed it 😀